.github/workflows/deps_lint.yml

```diff
@@ -87,6 +87,31 @@ jobs:
         .ci/openapi_diff.sh
         .ci/index_openapi_diff.sh
 
+  docker-image:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Build the Docker image
+      run: docker-compose build
+
+    - name: Test server
+      run: |
+        docker-compose up optimade &
+        .github/workflows/wait_for_it.sh localhost:3213 -t 120
+        sleep 15
+        curl http://localhost:3213/optimade/info > info.json
```

Comment:
This should now be the base url right? The fact that this works without it (because example.com is also included in the meta for a failed response) means we should also change the test case.

Reply:
I'm a little bit worried about how we're going to keep all these versioned base URLs up to date across the package, I thought that having the unversioned url point to the latest was quite nice for clients.

Reply:
> I'm a little bit worried about how we're going to keep all these versioned base URLs up to date across the package, I thought that having the unversioned url point to the latest was quite nice for clients.

Agree. But since the major version should only ever change for backwards-incompatible changes, it's fine, in my opinion, since we would have to go through everything quite thoroughly at that stage. Also, it should be a matter of search-and-replace `v0` with `v1`?

Reply:
> This should now be the base url right? The fact that this works without it (because example.com is also included in the meta for a failed response) means we should also change the test case.

Yup, will update this. Already have an updated one in my own implementation for AiiDA, since I actually ran into this exact issue - good catch!